src/components/App/App.js

```diff
@@ -32,9 +40,31 @@ class App extends React.Component {
 		}
 	};
 
+	showNotification = message => {
+		this.setState(
+			{notificationMessage: message, showNotification: true},
+			() => {
+				setTimeout(() => {
```

Comment:
Since we're not using the updated version of state in the callback could we do this separately?

e.g.

```
showNotification = notificationMessage => {
  this.setState({
    notificationMessage,
    showNotification: true,
  })

  setTimeout(() => {
    this.setState({ showNotification: false })
  }, 5000)
}
```